### DISCUSSION TITLE
PVE nag not disabled on mobile after post-install script

### DISCUSSION THREAD

**Discussion by skyl-issue** [tteck/Proxmox#3304] [Help Needed] [answered]
Hello all,

I'm having a problem accessing my homelab PVE server on mobile. The post-install script successfully disabled the subscription nag on the desktop web GUI but on iOS Firefox and Chrome the nag remains and additionally is unable to be removed by pressing the dismiss button on the nag making the mobile GUI completely unusable. 

I'm new to Proxmox and unable to afford a subscription so any help would be greatly appreciated.

**Comment by tteck** [comment]
Clear browser cache

**Comment by skyl-issue** [comment, marked as answer]
I solved my problem by requesting the desktop site instead of the mobile site. The nag was disabled on the desktop site and when I accessed the mobile version of the site after that the nag was removed there too. No browse cache clearing required. Thanks @tteck for the response and thanks for all the helpful scripts.
